### PR TITLE
Feature/den 2525 refactor price handling

### PR DESCRIPTION
**Pull request by migo315**

### CODE REVIEW

**Comment by migo315** [comment]
A interesting change: Instead of adding, changing and removing prices by ourselves, we just setting the _current_ prices and Commercetools will refresh this.

File: src/ActionBuilder/Product/SetPrices.php [hunk 1]
```diff
@@ -53,33 +64,13 @@ public function createUpdateActions(
...
-                $actions[] = ProductChangePriceAction::ofPriceIdAndPrice(
-                    $priceId,
-                    PriceDraft::fromArray($foundPrice->toArray())
-                )->setStaged($dataId === 'staged');
+        $priceDrafts = [];
+        foreach ($variant->getPrices() as $price) {
+            if ($price instanceof Price) {
+                $priceDrafts[] = PriceDraft::fromArray($price->toArray());
             }
         }
```
Comment (avareldev) [inline src/ActionBuilder/Product/SetPrices.php:73]: Mabye add a comment why this is enough

**Review by avareldev — DISMISSED** [review]

File: src/ActionBuilder/Product/SetPrices.php (hunk 1 above)
Comment (migo315) [reply src/ActionBuilder/Product/SetPrices.php:77]: I've added a comment.

**Review by avareldev — APPROVED** [review]

**Review by TheDonMonte — APPROVED** [review]
Didn't test

**Pull request merged**